src/kibiutils.js

```diff
@@ -167,12 +167,12 @@
     //
 
     const DatasourceTypes = {
-      sqlite: 'sqlite',
-      rest: 'rest',
-      postgresql: 'postgresql',
-      mysql: 'mysql',
+      sqlite:      'sqlite',
+      rest:        'rest',
+      postgresql:  'postgresql',
+      mysql:       'mysql',
       sparql_http: 'sparql_http',
-      sql_jdbc: 'sql_jdbc',
+      sql_jdbc:    'sql_jdbc',
```

Comment:
Somebody seem to have added this shitty rule. Do we really want this formatting? @szydan